pkg/telemetry/prometheus/node.go

```diff
@@ -160,118 +147,119 @@ func GetUpdatedNodeStats(prev *livekit.NodeStats, prevAverage *livekit.NodeStats
 		memUsed = memInfo.Used
 	}
 
-	// do not error out, and use the information if it is available
 	sysPackets, sysDroppedPackets, _ := getTCStats()
 	promSysPacketGauge.WithLabelValues("out").Set(float64(sysPackets - sysPacketsStart))
 	promSysPacketGauge.WithLabelValues("dropped").Set(float64(sysDroppedPackets - sysDroppedPacketsStart))
```

Comment:
Unrelated to this PR, but don't necessarily like this. The prom counters are getting updated as a side effect of some other module calling to get updated stats. But, not changing that behaviour in this PR.